app/assets/stylesheets/layouts/_sitemap.scss

```diff
@@ -0,0 +1,135 @@
+.sitemap {
+  padding: 0 15px;
+
+  @media screen and (min-width: $br-mobile) {
+    padding: 0;
+  }
+
+  header {
+    padding-top: 105px;
+    @media screen and (min-width: $br-mobile) {
+      padding-top: 85px;
+    }
+  }
+
+  header > h2 {
+    font-size: 47px;
+    color: $dark;
+    letter-spacing: -.7px;
+  }
+
+  .category-title {
+    margin-top: 75px;
+    font-size: 35px;
+    font-weight: 300;
+    line-height: 1.1;
+    letter-spacing: .2px;
+  }
+
+  .page {
+    padding: 25px 0;
+
+    border-bottom: 1px solid $border;
+
+    @media screen and (min-width: $br-mobile) {
+      padding: 45px 0;
+    }
+
+    &:last-child {
+      border-bottom: 0;
+    }
+  }
+
+  .page-title {
+    margin-bottom: 15px;
+
+    font-size: 21px;
+    line-height: 1.4;
+    letter-spacing: .1px;
+    text-transform: uppercase;
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .page-section-list {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .page-section-list > .page-section {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    margin-bottom: 15px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @media screen and (min-width: $br-mobile) {
+      flex-basis: 25%;
+      margin-bottom: 0;
+
+      &:nth-child(5n) {
+        margin-top: 15px;
+      }
+    }
+  }
+
+  .page-section > .title-section {
+    display: inline-block;
+    margin-bottom: 15px;
+
+    > a {
+      font-size: 19px;
+      line-height: 1.7px;
+      font-weight: 300;
+      color: $cGreen;
+    }
+
+    &:hover {
```

Comment:
It will better if we put the hover in the "a" tag, as long as the "li" is not going to take us anywhere. The same with the other ones